lib/friends/event.rb

```diff
@@ -132,9 +133,21 @@ def update_location_name(old_name:, new_name:)
     end
 
     # @param location [Location] the location to test
-    # @return [Boolean] true iff this activity includes the given location
+    # @return [Boolean] true if activity has location in description or it equals implicit location
     def includes_location?(location)
-      @description.scan(/(?<=_)[^_]+(?=_)/).include? location.name
+      location_in_description?(location) || location_is_implicit?(location)
+    end
+
+    def location_in_description?(location)
```

Comment:
Can this method be private?

Reply:
Yup! I think `location_is_implicit?` can be made private too

Reply:
Done